test/functional/apps/dashboard/group3/dashboard_state.ts

```diff
@@ -170,10 +170,9 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
     const getUrlFromShare = async () => {
       log.debug(`getUrlFromShare`);
       await PageObjects.share.clickShareTopNavButton();
-      const sharedUrl = await PageObjects.share.getSharedUrl();
+      const snapshotUrl = await PageObjects.share.getSnapshotUrl();
```

Comment:
Use the raw dashboard snapshot url to enable testing dashboard state, in versions > 8.15 this test suite is skipped hence why this change is not included in the initial PR